qt/scientific_interfaces/ISISReflectometry/Common/Clipboard.cpp

```diff
@@ -41,7 +42,8 @@ bool Clipboard::isGroupLocation(int rootIndex) const {
     throw std::runtime_error("Attempted to access invalid value in clipboard");
 
   // Check if the root is a group
-  if (!MantidQt::CustomInterfaces::isGroupLocation(subtreeRoots()[rootIndex]))
+  if (!MantidQt::CustomInterfaces::ISISReflectometry::isGroupLocation(
```

Comment:
You shouldn't need the `MantidQt::CustomInterfaces::ISISReflectometry::` here since it's already in that namespace.